src/app/views/change-password/changePasswordConfirmed.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Panel from "../../components/panel/Panel";
+
+const propTypes = {
+    validationErrors: PropTypes.shape({
+        hading: PropTypes.string,
+        body: PropTypes.oneOfType([PropTypes.string, PropTypes.element])
+    }),
+    onSubmit: PropTypes.func,
+    emailInput: PropTypes.shape({
+        id: PropTypes.string,
+        label: PropTypes.string,
+        type: PropTypes.string,
+        onChange: PropTypes.func,
+        error: PropTypes.string
+    })
+};
+
+const ChangePasswordConfirmed = props => {
+    const changePasswordPanelBody = (
+        <p>
+            If you have technical problems with your account, please email
```

Comment:
I can't quite tell from looking at the code (will need to run it to test) but think there is a missing space before the anchor tag. This is needed or we will see:
`If you have technical problems with your account, please emailpublishing@ons.gov.uk`
rather than 
`If you have technical problems with your account, please email publishing@ons.gov.uk`